modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,136 @@
+
+. Log in to Anypoint Platform.
+. Navigate to your user profile.
+. Click *Configure multi-factor authentication (MFA)*. +
+The *Manage Your Verification Methods* interface appears.
+. Next to the verification method you want to configure, click *Add*.
+. Follow the instructions for your preferred verification method.
+
+== Modify or Remove Multi-factor Authentication on Individual User Accounts
+
+If you have previously configured multi-factor authentication, you can modify which verification methods you use for your account. If your organization requires multi-factor authentication, you must have at least one verification method configured.
+
+If your organization has disabled its multi-factor authentication requirement, you can also remove all verification methods. However, for enhanced security, it is best to use at least one verification method.
+
+To modify verification methods on an individual user account:
+
+. Log in to Anypoint Platform.
+. Navigate to your user profile.
+. Click *Configure multi-factor authentication (MFA)*. +
+The *Manage Your Verification Methods* interface appears.
+. Add an additional verification method.
+.. Next to the verification method you want to configure, click *Add*.
+.. Follow the instructions to configure your preferred verification method.
+.. Click *Done*. +
+You can now use the configured verification method to verify your identity when you log in to Anypoint Platform.
+. Modify the name of an existing verification method.
+.. Click the pencil icon next to the verification method that you want to rename.
+.. Enter a new name for your verification method.
+.. Click the checkmark.
+.. Click *Done*. +
+The name of the verification method is changed.
+. Remove a verification method.
+.. Next to the verification method you want to remove, click the bin icon.
+.. Click *Yes*
+.. Click *Done*. +
+The verification method is removed. If your organization requires multi-factor authentication and you have removed all verification methods, you must configure at least one verification method before you save your changes.
+. Click *Save*.
+
+== Disable Multi-factor Authentication Requirement for All User Accounts
+
+When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users must manually remove multi-factor authentication in their profile settings.
+
+To disable multi-factor authentication for all users:
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor auth*.
+. Unselect *Required*.
+. Click *Save*. +
+Multi-factor authentication configuration is no longer required for new users.
```

Comment:
Do we really need to confirm that the goal of the task is reached after the final step?

Reply:
This is a big deal for an org.